definitions/features/pulp2.rb

```diff
@@ -5,7 +5,9 @@ class Features::Pulp < ForemanMaintain::Feature
     label :pulp2
 
     confine do
-      find_package('pulp-server')
+      if check_max_version('katello-common', '3.18')
+        find_package('pulp-server')
+      end
```

Comment:
Isn't this the same?
```suggestion
      check_max_version('katello-common', '3.18') && find_package('pulp-server')
```

Or to make the final diff easier to read:
```suggestion
      find_package('pulp-server') && check_max_version('katello-common', '3.18')
```

Reply:
I like it.